packages/app-desktop/gui/PasswordInput/PasswordStrengthIndicator.js

```
@@ -0,0 +1,43 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const React = require("react");
+const locale_1 = require("@joplin/lib/locale");
+const PasswordStrengthIndicator = (props) => {
+    const { password } = props;
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)